tests/acceptance/features/apiOcm/share.feature

```diff
@@ -725,11 +725,30 @@ Feature: an user shares resources using ScienceMesh application
     And "Brian" has accepted invitation
     And user "Brian" has created a folder "FOLDER" in space "Personal"
     And user "Brian" has sent the following resource share invitation to federated user:
-      | resource        | FOLDER                        |
-      | space           | Personal                      |
-      | sharee          | Alice                         |
-      | shareType       | user                          |
-      | permissionsRole | Editor                        |
+      | resource        | FOLDER   |
+      | space           | Personal |
+      | sharee          | Alice    |
+      | shareType       | user     |
+      | permissionsRole | Editor   |
     When using server "LOCAL"
     And user "Alice" uploads a file with content "lorem" to "file.txt" inside federated share "FOLDER" via TUS using the WebDAV API
     Then for user "Alice" the content of file "file.txt" of federated share "FOLDER" should be "lorem"
+
+  @issue-10495
+  Scenario: local user downloads thumbnail preview of a federated shared image
+    Given using spaces DAV path
+    And using server "LOCAL"
+    And "Alice" has created the federation share invitation
+    And using server "REMOTE"
+    And "Brian" has accepted invitation
+    And user "Brian" has uploaded file "filesForUpload/testavatar.jpg" to "testavatar.jpg"
+    And user "Brian" has sent the following resource share invitation to federated user:
+      | resource        | testavatar.jpg |
+      | space           | Personal       |
+      | sharee          | Alice          |
+      | shareType       | user           |
+      | permissionsRole | Viewer         |
+    And using server "LOCAL"
+    When user "Alice" downloads the preview of federated share image "testavatar.jpg" with width "32" and height "32" and processor "thumbnail" using the WebDAV API
```

Comment:
Following the same approach as the previous steps :
``` feature
When user "Alice" downloads the preview of "/testimage.txt" with width "32" and height "32" using the WebDAV API
When user "Brian" downloads the preview of shared resource "/Shares/<file-name>" with width "32" and height "32" using the WebDAV API
When user "Alice" downloads the preview of "<destination>" of the space "previews of the files" with width "<width>" and height "<height>" using the WebDAV API
```
We may or may not add more tests with different image dimensions. Very uncertain.
I think it's better to dynamically give dimensions in the steps rather than setting a value in the step-definition.

Reply:
Actually, we can download without the dimension as well but as i mentioned, just following same approach.
We are checking the content.
```And the downloaded preview content should match with "federated-share-thumbnail.png" fixtures preview content```

Reply:
>The question is, does dimension matter? We check downloaded image dimension or not?

yes, dimension matters in this test. (we download 32x32 and check if it is 32x32)

Please, use this previous step. Because it is not very good to hide the import details of a request/action
```feature
When user "Alice" downloads the preview of federated share image "testavatar.jpg" with width "32" and height "32" and processor "thumbnail" using the WebDAV API
```

but you can remove `processor "thumbnail"` from the step or even do not request with processor param. It can always be `thumbnail` (default)